modules/acquia_cms_article/tests/src/ExistingSite/ArticleListTest.php

```diff
@@ -25,8 +25,11 @@ class ArticleListTest extends ContentTypeListTestBase {
   /**
    * {@inheritdoc}
    */
-  protected function getView() : View {
-    return View::load('articles');
+  protected function getView() {
+    if ($this->container->get('module_handler')->moduleExists('acquia_cms_search')) {
+      return View::load('articles');
```

Comment:
I think for the scope of this PR, this is ok, but I'm concerned that we don't have an articles view in the absence of the search module